File: wordle-backend/Models/UserModel.cs
using System.ComponentModel.DataAnnotations;

namespace Models
{
    public class UserModel
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        [Required]
        public string? EmailId { get; set; }
        [Required]
        public string? Password { get; set; }
        [Key]
        public string? GUID { get; set; }
    }
}

File: wordle-client/src/app/Models/UserModel.ts
export class UserModel {
    public name: string;
    public emailId: string;
    public password: string;

    constructor() {
        this.name = "";
        this.emailId = "";
        this.password = "";
    }

    import(name: string | null, email: string | null, password: string | null) {
        this.name = name ? name : "";
        this.emailId = email ? email : "";
        this.password = password ? password : "";
    }
}